src/creature.cpp

```diff
@@ -990,6 +990,21 @@ void Creature::process_effects()
     }
 }
 
+bool Creature::resists_effect(effect e)
```

Comment:
The function should probably be const, it does not change the creature at all.

The effect does not need to be copied and should be a const reference.